### ISSUE TITLE
Request for Feedback: Root Link Relation Update

### ISSUE THREAD

**Issue by mdobson** [zettajs/zetta#38]
Hey I'm going through and building a small API consumer app before the conference, and just making notes of things that I observe when building. I'd like to add a server relation to all the peer links in addition to the peer relation. I think it would make it easier for a client to access all the relevant servers quickly with this relation in place. Thoughts? 

```json
{
  "class": [
    "root"
  ],
  "links": [
    {
      "rel": [
        "self"
      ],
      "href": "http://zetta-cloud-2.herokuapp.com/"
    },
    {
      "title": "cloud",
      "rel": [
        "http://rels.zettajs.io/server"
      ],
      "href": "http://zetta-cloud-2.herokuapp.com/servers/2d72035a-fff5-4183-a4da-ca6001706b76"
    },
    {
      "title": "matt.dobson",
      "rel": [
        "http://rels.zettajs.io/peer",
        "http://rels.zettajs.io/server"
      ],
      "href": "http://zetta-cloud-2.herokuapp.com/servers/5961a387-44ee-4e8a-8d88-9791fcb6b0fc"
    },
    {
      "title": "Detroit",
      "rel": [
        "http://rels.zettajs.io/peer",
        "http://rels.zettajs.io/server"
      ],
      "href": "http://zetta-cloud-2.herokuapp.com/servers/d23094cf-aba5-4c7b-b3ee-b43807ce4e95"
    },
    {
      "rel": [
        "http://rels.zettajs.io/peer-management"
      ],
      "href": "http://zetta-cloud-2.herokuapp.com/peer-management"
    }
  ]
}
```

**Comment by kevinswiber** [comment]
I can get behind this.

**Comment by kevinswiber** [comment]
Fixed by #188.
